src/rgw/rgw_admin.cc

```diff
@@ -98,7 +98,7 @@ void usage()
   cout << "  key create                 create access key\n";
   cout << "  key rm                     remove access key\n";
   cout << "  bucket list                list buckets (specify --allow-unordered for\n";
-  cout << "                             faster, unsorted listing)\n";
+  cout << "                             faster, unsorted listing; --extra-info for manifest detail)\n";
```

Comment:
welcome to the club!

Reply:
the make-check-bot will compare this against src/test/cli/radosgw-admin/help.t and flag an error. can you please update help.t to match?

Reply:
indeed, you paved the way, eric :)